Assets/Scenes/TestScene.unity

```diff
@@ -202,7 +202,12 @@ MonoBehaviour:
   displayText: Delicious Cube
   displaySubtext: '[E] to pick up'
   ignoreTagCheck: 0
+<<<<<<< HEAD
+  itemToPickup: {fileID: 58653346}
+  inventory: {fileID: 143817151}
+=======
   pickupItem: {fileID: 58653346}
+>>>>>>> origin/master
```

Comment:
Merge conflict unsolved?

Reply:
Nothing at my end. So I don't know.

Reply:
I did this with the tool and Github is not showing conflicts and I don't  to fix i

Reply:
![grafik](https://user-images.githubusercontent.com/24923925/79513761-e49f5480-8044-11ea-89d7-1b9212f7cec2.png)

I still cannot open the scene, there are conflicts

Reply:
I did some more research and the UnityYAML Merger didn't do a thing. So can some one with a working version do it for me?